packer/build_image.sh

```diff
@@ -304,6 +304,9 @@ IMAGE_NAME="$PRODUCT-$VERSION-$ARCH-$(date '+%FT%T')"
 if [ "$BUILD_MODE" = "debug" ]; then
   IMAGE_NAME="$PRODUCT-debug-$VERSION-$ARCH-$(date '+%FT%T')"
 fi
+if [ "$TARGET" = "azure" ]; then
+  IMAGE_NAME="scylla-$VERSION-$ARCH-$(date '+%FT%T')"
```

Comment:
this mean that even debug versions would get the same names ? 

since we don't have any tagging, and selection by tagging in Azure, that mean that all azure builds would be named the same...

Reply:
no, see line 310:
```
if $DEBUG ; then
  IMAGE_NAME="debug-$IMAGE_NAME"
fi
```

Reply:
@fruch The first one is referring to `build mode`